internal/subshell/sscommon/sscommon.go

```diff
@@ -177,33 +175,13 @@ func binaryPathCmd(env []string, name string) (string, error) {
 	return split[0], nil
 }
 
-func ignoreInterrupts(ctx context.Context) {
```

Comment:
I don't think we should be removing this. IIRC it is required so as not to bubble stuff up from the subshell context. For this story we're concerned with interrupts happening outside of subshell.